tests/scanner/groups_scanner_test.py

```diff
@@ -40,9 +40,21 @@ def _render_ascii(self, starting_node, attr):
         Returns:
             attr: String of the attribute to render.
         """
-        return anytree.RenderTree(
-            starting_node,
-            style=anytree.AsciiStyle()).by_attr(attr)
+        rows = []
+        for pre, fill, node in anytree.RenderTree(starting_node,
+                                                  style=anytree.AsciiStyle()):
+            value = getattr(node, attr, "")
+            if isinstance(value, (list, tuple)):
+                lines = value
+            else:
+                lines = str(value).split("\n")
+            rows.append(u"%s%s" % (pre,
+                                   json.dumps(lines[0], sort_keys=True)))
+            for line in lines[1:]:
+                rows.append(u"%s%s" % (fill,
+                                       json.dumps(line, sort_keys=True)))
+
+        return str('\n'.join(rows))
```

Comment:
nit: join method returns a string, the str() is not needed

Reply:
Good catch, left over from testing.